src/dyad/core/dyad_core.c

```diff
@@ -303,6 +303,10 @@ DYAD_CORE_FUNC_MODS dyad_rc_t dyad_fetch (const dyad_ctx_t* restrict ctx,
     char upath[PATH_MAX] = {'\0'};
     const size_t topic_len = PATH_MAX;
     char topic[PATH_MAX + 1] = {'\0'};
+    if (ctx->shared_storage) {
+        rc = DYAD_RC_OK;
```

Comment:
I did not add `DYAD_C_FUNCTION_UPDATE_INT ("is_local", 1);` here to avoid adding obvious information while increasing the log size.

Reply:
I think it useful to have to easily filter those out. :)

Reply:
When shared_storage is on all accesses are going to be treated as local. Would it be useful to have a single message at the beginning, which we may already have, about shared storage?